lib/u3d/commands.rb

```diff
@@ -110,6 +110,7 @@ def download(args: [], options: {})
 
         unity = check_unity_presence(version: version)
         return unless enforce_setup_coherence(packages, options, unity)
+        return if packages.empty?
```

Comment:
I am not sure this is what we want. Empty packages mean install everything as well. See `Downloader.download_modules`